electrician/demo/stock_quant.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="stock_quant_1" model="stock.quant">
```

Comment:
Does it make sense to define stock.quant records without calling `action_apply_inventory`? From what I see, this file can be removed (to check)